app/repositories/guide_repository.rb

```diff
@@ -11,6 +11,10 @@ def self.slugs
     @slugs ||= new.slugs
   end
 
+  def self.cacheable_slugs
```

Comment:
We needed to strip the question pages here so cookies are stripped and flash works.